superset-frontend/src/explore/components/controls/SelectControl.test.jsx

```diff
@@ -37,9 +37,9 @@ const defaultProps = {
 };
 
 const options = [
-  { value: '1 year ago', label: '1 year ago', order: 0 },
-  { value: '1 week ago', label: '1 week ago', order: 1 },
-  { value: 'today', label: 'today', order: 2 },
```

Comment:
May do a bigger sweep in other components to clean up similar `order` props later.